packages/dialog/src/__tests__/Dialog.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import { Dialog, DialogContent, DialogFooter } from '../'
+import { Text, Button } from '@kodiak-ui/primitives'
+
+function DialogExample() {
+  const [isOpen, setIsOpen] = React.useState(false)
+
+  return (
+    <>
+      <div data-testid="outside">
+        Some content outside to click on to close the modal
+      </div>
+      <Button onClick={() => setIsOpen(true)}>Show default example</Button>
+      <Dialog
+        isOpen={isOpen}
+        title={
+          <div>
+            <Text as="h3" mb={2}>
+              Testing
+            </Text>
+            <Text mb={0}>Testing some text</Text>
+          </div>
+        }
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onDismiss={() => setIsOpen(false)}
+        aria-label="Warning about next steps"
+      >
+        <DialogContent>
+          <Text as="p">
```

Comment:
Text starts out as `p` so probably don't need to specify.

Reply:
Yep, good call.